plugins/modules/identity/ipa/ipa_config.py

```diff
@@ -22,25 +22,141 @@
     description: Default e-mail domain for new users.
     aliases: ["emaildomain"]
     type: str
+  ipadefaultprimarygroup:
+    description: Default group for new users.
+    aliases: ["primarygroup"]
+    type: str
+    version_added: '2.4.0'
+  ipagroupsearchfields:
+    description: A comma-separated list of fields to search in when searching for groups.
+    aliases: ["groupsearchfields"]
+    type: str
+    version_added: '2.4.0'
+  ipahomesrootdir:
+    description: Default location of home directories.
+    aliases: ["homesrootdir"]
+    type: str
+    version_added: '2.4.0'
+  ipamaxusernamelength:
+    description: Maximum length of usernames.
+    aliases: ["maxusernamelength"]
+    type: str
+    version_added: '2.4.0'
+  ipapwdexpadvnotify:
+    description: Notice of impending password expiration, in days.
+    aliases: ["pwdexpadvnotify"]
+    type: str
+    version_added: '2.4.0'
+  ipasearchrecordslimit:
+    description: Maximum number of records to search (-1 or 0 is unlimited).
+    aliases: ["searchrecordslimit"]
+    type: str
+    version_added: '2.4.0'
+  ipasearchtimelimit:
+    description: Maximum amount of time (seconds) for a search (-1 or 0 is unlimited).
+    aliases: ["searchtimelimit"]
+    type: str
+    version_added: '2.4.0'
+  ipauserauthtype:
+    description: The authentication type to use by default.
+    aliases: ["userauthtype"]
+    choices: ["password", "radius", "otp", "pkinit", "hardened", "disabled"]
+    type: str
+    version_added: '2.4.0'
+  ipausersearchfields:
+    description: A comma-separated list of fields to search in when searching for users.
+    aliases: ["usersearchfields"]
+    type: str
+    version_added: '2.4.0'
 extends_documentation_fragment:
 - community.general.ipa.documentation
 
 '''
 
 EXAMPLES = r'''
-- name: Ensure the default login shell is bash.
+- name: Ensure the default login shell is bash
   community.general.ipa_config:
     ipadefaultloginshell: /bin/bash
     ipa_host: localhost
     ipa_user: admin
     ipa_pass: supersecret
 
-- name: Ensure the default e-mail domain is ansible.com.
+- name: Ensure the default e-mail domain is ansible.com
+  community.general.ipa_config:
+    ipadefaultemaildomain: ansible.com
+    ipa_host: localhost
+    ipa_user: admin
+    ipa_pass: supersecret
+
+- name: Ensure the default e-mail domain is ansible.com
```

Comment:
Maybe I'm misreading something, but isn't this example now present twice in a row?

Reply:
Addressed in 060cd1f.